azure-pipelines.yml

```diff
@@ -10,13 +10,18 @@ pr:
 jobs:
 - job: 'Test'
   pool:
-    vmImage: 'ubuntu-latest'
-
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6.9'
+      Python37:
+        python.version: '3.7.4'
   steps:
   - task: UsePythonVersion@0
     inputs:
-      versionSpec: '3.7'
-    displayName: 'Set python version to 3.7'
+      versionSpec: '${python.version}'
+    displayName: 'Set python version according to build matrix'
```

Comment:
You might also be able to use `$(python.version)` here to provide a prettier display name.